src/web/app/components/question-edit-form/question-edit-form.component.html

```diff
@@ -59,26 +59,11 @@
       <div class="background-color-light-blue">
         <div class="row">
           <div class="col-12">
-            <div class="row padding-15px text-left">
-              <div class="col-md-2">
-                <div class="question-basic-info">Question</div>
-              </div>
-              <div class="col-md-10">
-                <textarea id="question-brief" class="form-control" [ngModel]="model.questionBrief" (ngModelChange)="triggerModelChange('questionBrief', $event)" [disabled]="!model.isEditable"
-                          placeholder="A concise version of the question e.g. &quot;How well did the team member communicate?&quot;"
-                ></textarea>
-              </div>
-            </div>
-            <div class="row padding-15px text-left">
-              <div class="col-md-2">
-                <div class="question-basic-info">[Optional]<br/>Description</div>
-              </div>
-              <div class="col-md-10">
-                <tm-rich-text-editor [richText]="model.questionDescription" (richTextChange)="triggerModelChange('questionDescription', $event)" [isDisabled]="!model.isEditable" minHeightInPx="100"
-                                     placeholderText="More details about the question e.g. &quot;In answering the question, do consider communications made informally within the team, and formal communications with the instructors and tutors.&quot;"
-                ></tm-rich-text-editor>
-              </div>
-            </div>
+            <tm-question-edit-brief-description-form [brief]="model.questionBrief" (briefChange)="triggerModelChange('questionBrief', $event)" [isBriefDisabled]="!model.isEditable"
+                                                     briefPlaceholder="A concise version of the question e.g. &quot;How well did the team member communicate?&quot;"
+                                                     [descriptionRichText]="model.questionDescription" (descriptionChange)="triggerModelChange('questionDescription', $event)" [isDescriptionDisabled]="!model.isEditable"
```

Comment:
`descriptionRichText` is slightly too wordy. Just `description` should be sufficient.

Reply:
I can make the change, but it's named `descriptionRichText` because in this way we can emphasize it's rich text instead of normal string.

Reply:
It actually took me a while to figure out how it can display text with styles before I noticed it was stored as rich text.

Reply:
"Description being rich text" is an implementation detail within the component itself (in this case, `tm-question-edit-brief-description-form`) that the caller (in this case, `tm-question-edit-form`) does not really need to care about. We do not see `@Input`s being named `*Number`, `*String`, `*[insert custom class name here]` etc.

And within the component `tm-question-edit-brief-description-form`, it is immediately obvious by the fact that the string is rendered within a `tm-rich-text-editor` component.

Reply:
Okay I understand your rationale. I'll stick to the convention here.